gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb

```diff
@@ -1,842 +1,803 @@
 {
```

Comment:
<p>Naming of the section title and model names. use "candidate model" "baseline model", "Gemini-1.0-Pro", also "Gen AI Eval Service SDK"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
This is not autosxs example. not update to candidate model and baseline model